src/commands/session.ts

```
@@ -13,18 +13,44 @@ import {
+		},
+	},
+	toKebab: true,
 	async run(ctx) {
 		// --jq implies --json
 		const useJson = ctx.values.json || ctx.values.jq != null;
 		if (useJson) {
 			logger.level = 0;
 		}
 
+		// Handle specific session ID lookup
+		if (ctx.values.id != null) {
```

ESLint

[error] Unsafe member access .values on an `any` value.

(ts/no-unsafe-member-access)